Assets/Scripts/PlayerControls.cs

```diff
@@ -188,7 +287,7 @@ bool RaycastForward( out RaycastHit closestHit )
 			new Ray( camPos, camForward ),
```

Comment:
Consider a more descriptive name for this method. CheckForHitInteractable maybe?